timelines_from_iuns/README.md

```diff
@@ -0,0 +1,15 @@
+# read a list of IUNs from a TXT file and downloads the timelines from DynamoDB, writing to JSON.file
```

Comment:
aggiungiamo alla fine del readme in che modo deve essere formattato il file per darlo in input correttamente 
giusto un paio di commenti del tipo:
TUQU-NUKZ-XJAR-202310-U-1
PLWA-ZMDA-MDJE-202310-X-1
QJWD-NEJR-TJEX-202310-Y-1